lib/rest-api.php

```diff
@@ -338,6 +338,48 @@ function gutenberg_add_target_schema_to_links( $response, $post, $request ) {
 	return $response;
 }
 
+/**
+ * Include revisions data on post response links.
+ *
+ * @see https://core.trac.wordpress.org/ticket/44321
+ *
+ * @param WP_REST_Response $response WP REST API response of a post.
+ * @param WP_Post          $post The post being returned.
+ * @param WP_REST_Request  $request WP REST API request.
+ * @return WP_REST_Response Response containing the new links.
+ */
+function gutenberg_add_revisions_data_to_links( $response, $post, $request ) {
+
+	$new_links  = array();
+	$orig_links = $response->get_links();
+
+	if ( ! empty( $orig_links['version-history'] ) ) {
+		$version_history_link = array_shift( $orig_links['version-history'] );
+		// 'version-history' already exists and we don't want to duplicate it.
+		$response->remove_link( 'version-history' );
+
+		$revisions       = wp_get_post_revisions( $post->ID, array( 'fields' => 'ids' ) );
+		$revisions_count = count( $revisions );
+
+		$new_links['version-history'] = array(
+			'href'  => $version_history_link['href'],
+			'count' => $revisions_count,
+		);
+
+		if ( $revisions_count > 0 ) {
```

Comment:
Shouldn't it be greater than 1? I think we don't render the link if there is only 1 revision.

Yep, confirmed: https://github.com/WordPress/gutenberg/blob/8c5678c4735759263c7cc25ceb0d51cc30be3fd7/editor/components/post-last-revision/check.js#L13

Reply:
>Shouldn't it be greater than 1? I think we don't render the link if there is only 1 revision.

That's a client specific detail. Seems to me the REST API should provide the raw value (maybe even if it were `0`).

Reply:
Assuming it is still a correct url, is fine as is.